owncloudComLibrary/src/main/java/com/owncloud/android/lib/resources/shares/ShareParserResult.kt

```diff
@@ -27,4 +27,4 @@ package com.owncloud.android.lib.resources.shares
 
 import java.util.ArrayList
 
-class ShareParserResult(val shares: ArrayList<RemoteShare>, val parserMessage: String)
```

Comment:
Why don't you use List instead of ArrayList?? IMHO, it's always better to use the interface ;-)